lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp

```diff
@@ -385,7 +387,7 @@ PathTracker::getOrComputeNeedsAltBasePath(Location loc, StringAttr moduleName,
     }
     // If there is more than one instance of this module, then the path
     // operation is ambiguous, which is an error.
```

Comment:
Can the PR also update this comment which is now stale?

Reply:
Oops, yep will update.